packages/selinux-policy/rules.cil

```diff
@@ -26,6 +26,13 @@
 ; Unprivileged components cannot interact with privileged processes.
 (neverallow unprivileged_s privileged_s (processes (interact)))
 
+; Host subjects aren't granted the `noatsecure` permission to clear harmful
+; environment variables in automatic transitions. Since the transition is
+; successful, the host subjects will execute in the secure environment, and
+; containers' processes are given the relax treatment, it is safe to suppress
+; this audit message.
```

Comment:
nit: maybe just this:
```suggestion
; Host subjects are denied the `noatsecure` permission to ensure that
; the environment is sanitized during all domain transitions. Suppress
; this audit message since otherwise it adds a lot of noise.
```